notebooks/3_KernelMethods.ipynb

```diff
@@ -1173,7 +1168,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "kpca.fit(X_train, K=K_train)"
+    "kpca.fit(X_train)"
```

Comment:
see earlier comment

Reply:
If  I use `**kernel_params` as said in the comment on line 630, I can‘t set `kernel = 'precomputed'` simultaneously.  So I use `**kernel_params` here but add a comment to remind readers that they can modify it if they want.
![image](https://user-images.githubusercontent.com/56018055/99543674-b6f00480-29b3-11eb-9c48-2aebd822f70c.png)

Reply:
then use `kernel_params = dict(kernel="precomputed")`!

Reply:
I don't think it's a good idea. If we use `kernel_params = dict (kernel= "precomputed")`, this means that we use `kpca.fit (X = K_train)` throughout the notebook. If readers want to use other kernel functions, they need to change `kpca.fit (X = K_train)` to `kpca.fit (X = X_train)` many times. This is contrary to the purpose of using `kernel_params`.
![image](https://user-images.githubusercontent.com/56018055/99810485-0ebd7580-2b44-11eb-9f82-33e43a1b4bbb.png)

Reply:
Also, using `kernel_params = dict (kernel= "precomputed")` doesn't help save much time while running the notebook. I don't quite understand why it must be used.

Reply:
it's not only about time, but about RAM. We have to assume that these notebooks will be used in a pedagogic setting, where students will be using computers of a variety of capabilities. It may be fast on your computer, but if we _know_ it's a potential time-sink, we may as well avoid it.